seqr/views/apis/summary_data_api_tests.py

```diff
@@ -701,12 +731,253 @@ def test_sample_metadata_export(self, mock_google_authenticated):
         response = self.client.get(f'{gregor_projects_url}?includeAirtable=true')
         self._has_expected_metadata_response(response, multi_project_individuals, has_airtable=True, has_duplicate=True)
 
+    def test_family_metadata(self):
+        url = reverse(family_metadata, args=['R0003_test'])
+        self.check_collaborator_login(url)
```

Comment:
copy pasted from report_api_tests, only change is changing this from `check_analyst_login` and then adding analyst login back in later